test/extensions/filters/http/ext_proc/ext_proc_integration_test.cc

```diff
@@ -1599,7 +1599,8 @@ TEST_P(ExtProcIntegrationTest, GetAndSetHeadersAndTrailersOnResponse) {
 // Test the filter using the default configuration by connecting to
 // an ext_proc server that tries to modify the trailers incorrectly
 // according to the header mutation rules.
-TEST_P(ExtProcIntegrationTest, GetAndSetTrailersIncorrectlyOnResponse) {
+#TODO(@tyxia): reenable this test (see https://github.com/envoyproxy/envoy/issues/35281)
```

Comment:
Do you just need `TODO(tyxia)` ?

At least that is what i used in the past :)

Reply:
probs that would have worked - hoping the format issue is resolved now tho

Reply:
argh - still breaking 8(

Reply:
could just do `TODO(tyxia) see #35281`

Reply:
i think the problem was i was using wrong comment prefix - ie `#` and not `//`